src/marketplaces/sg.ts

```diff
@@ -14,5 +14,6 @@ export const SG = new AmazonMarketplace({
   webServiceUri: 'https://mws-fe.amazonservices.com',
   sellingPartner: {
     region: sellingPartnerRegions.FE,
+    sellerCentralUrl: sellerCentralUrls.NA,
```

Comment:
Are you sure that SG uses NA URIs?

This is actually not specified in the docs, but there were some questions about it from people I know :)

This is still not conclusive. We can probably merge it, but we need to create an issue for it to double check and update, if necessary.

Reply:
The docs just specify, that Spain, UK, France, Germany, and Italy use the sellercentral-europe endpoint. I think that means the other ones use the default sellercentral endpoint. Not conclusive though, yes.

Reply:
https://github.com/amzn/selling-partner-api-docs/issues/419

Reply:
Right, I know the docs say that. But I think the reality might be different. As I said, we can go ahead, but I am not certain this is right 😁